extensions/amp-base-carousel/1.0/base-carousel.js

```diff
@@ -319,6 +320,19 @@ function BaseCarouselWithRef(
       }}
       tabIndex="0"
       wrapperClassName={classes.carousel}
+      contentAs={lightbox ? WithLightbox : 'div'}
+      contentRef={contentRef}
+      contentProps={{
+        autoLightbox: false,
```

Comment:
Hmm. This is a bit confusing. `WithLightbox{autoLightbox: false}` simply renders itself as `{children}`. I think this would mean that `contentAs=WithLightbox` will actually skip rendering the content element entirely? This might also point to possibly better nested `WithLightbox` inside here, or maybe around slides themselves.

Reply:
Oh good catch. Updated this to always render the `Comp` layer.